openslides/utils/rest_api.py

```diff
@@ -225,15 +225,10 @@ def retrieve(self, request: Any, *args: Any, **kwargs: Any) -> Response:
             response = super().retrieve(request, *args, **kwargs)
         else:
             lookup_url_kwarg = self.lookup_url_kwarg or self.lookup_field
-            collection_element = CollectionElement.from_values(
```

Comment:
Is the docstring in line 214 still valid?

Reply:
no. I removed it